covid-19-support/src/components/Results.vue

```diff
@@ -74,10 +78,13 @@ export default {
     async fetchData(query) {
       try {
         // console.log(cartoBaseURL + '&q=' + query)
+        this.fetchDataState = StatusEnum.loading
         const res = await fetch(cartoBaseURL + '&q=' + query)
```

Comment:
fetch doesn't actually raise an error on a failed request. We have to manually raise an error.

We could do:
```
    const res = await fetch(cartoBaseURL + '&q=' + query)
    this.handle Errors(res)
```
And define handleErrors as:
```
    handleErrors(response) {
        if (!response.ok) {
            throw Error(response.statusText)
        }
    }
```